content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ ---
+reviewers:
+- caesarxuchao
+- dchen1107
+title: नोड्स
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes _Nodes_ पर चलने के लिए पॉड्स में कंटेनर रखकर आपका {{< glossary_tooltip text="वर्कलोड" term_id="workload" >}} चलाता है।
+क्लस्टर के आधार पर एक नोड वर्चुअल या भौतिक मशीन हो सकता है। प्रत्येक नोड
+द्वारा प्रबंधित किया जाता है
+{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}}
+और चलाने के लिए आवश्यक सेवाएं शामिल हैं
+{{< glossary_tooltip text="पॉड्स" term_id="pod" >}}.
+
+आम तौर पर आपके पास क्लस्टर में कई नोड होते हैं; एक सीखने या संसाधन-सीमित में
+पर्यावरण, आपके पास केवल एक नोड हो सकता है।
+
+नोड पर [घटक](/docs/concepts/overview/components/#node-components) में शामिल हैं
+{{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}},
+{{< glossary_tooltip text="कंटेनर रनटाइम" term_id="container-runtime" >}}, और
+{{< glossary_tooltip text="क्यूब-प्रॉक्सी" term_id="kube-proxy" >}}.
+
+<!-- body -->
+
+## प्रबंध
+
+{{< glossary_tooltip text="एपीआई सर्वर" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
+
+1. नोड पर क्यूबलेट नियंत्रण विमान में स्व-पंजीकृत होता है
+2. आप (या अन्य मानव उपयोगकर्ता) मैन्युअल रूप से नोड ऑब्जेक्ट जोड़ते हैं
+
+आपके द्वारा एक नोड बनाने के बाद {{< glossary_tooltip text="ऑब्जेक्ट" term_id="object" >}},
+या नोड पर क्यूबलेट स्व-पंजीकरण करता है, नियंत्रण विमान यह जांचता है कि नया नोड ऑब्जेक्ट है या नहीं
+वैध। उदाहरण के लिए, यदि आप निम्न JSON मेनिफेस्ट से नोड बनाने का प्रयास करते हैं:
+
+```json
+{
+  "kind": "Node",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "10.240.79.157",
+    "labels": {
+      "name": "my-first-k8s-node"
+    }
+  }
+}
+```
+
+कुबेरनेट्स आंतरिक रूप से एक नोड वस्तु बनाता है (प्रतिनिधित्व)। कुबेरनेट्स जाँच करता है
+कि एक क्यूबलेट एपीआई सर्वर पर पंजीकृत है जो `metadata.name` से मेल खाता है
+नोड का क्षेत्र। यदि नोड स्वस्थ है (यानी सभी आवश्यक सेवाएं चल रही हैं),
+तो यह पॉड चलाने के योग्य है। अन्यथा, उस नोड को किसी भी क्लस्टर गतिविधि के लिए अनदेखा कर दिया जाता है
+जब तक यह स्वस्थ न हो जाए।
+
+{{< note >}}
+कुबेरनेट्स ऑब्जेक्ट को अमान्य नोड के लिए रखता है और यह देखने के लिए जांच जारी रखता है कि क्या
+यह स्वस्थ हो जाता है।
+
+आपको, या किसी {{< glossary_tooltip term_id="कंट्रोलर" text="controller">}} को स्पष्ट रूप से
```

Comment:
The values of `term_id` and `text` parameters seem to be swapped. Please ensure that term_id is in English and text is in Hindi.

```suggestion
आपको, या किसी {{< glossary_tooltip term_id="controller" text="कंट्रोलर">}} को स्पष्ट रूप से
```